exercises/scale-generator/tests/scale-generator.rs

```diff
@@ -0,0 +1,179 @@
+//! Tests for scale-generator
+//!
+//! Generated by [script][script] using [canonical data][canonical-data]
+//!
+//! [script]: https://github.com/exercism/rust/blob/master/bin/init_exercise.py
+//! [canonical-data]: https://raw.githubusercontent.com/exercism/problem-specifications/master/exercises/scale-generator/canonical_data.json
+
+extern crate scale_generator;
+use scale_generator::*;
+
+/// Process a single test case for the property `chromatic`
```

Comment:
Hmm, I guess it's a bit unfortunate that both of these are `pitches` in the JSON file.

Also, potential confusion for students by seeing "property `chromatic`" here and "`pitches` property" two lines down? I'd consider making these two the same.

Same comment for `interval`